lib/awful/hotkeys_popup/widget.lua

```diff
@@ -371,6 +427,22 @@ function widget.new(args)
             beautiful.hotkeys_description_font or "Monospace 8"
         self.group_margin = args.group_margin or
             beautiful.hotkeys_group_margin or dpi(6)
+        self.highlight_bg = args.highlight_bg or
+            beautiful.hotkeys_highlight_bg or beautiful.bg_urgent
+        self.highlight_fg = args.highlight_fg or
+            beautiful.hotkeys_highlight_fg or beautiful.fg_urgent
+        self.find_prompt = args.find_prompt or
+            beautiful.hotkeys_find_prompt or "<b>Find: </b>"
```

Comment:
as i mentioned in previous comment this shouldn't be defined in beautiful, as its role defining theme, not translation/labels